webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/__tests__/components/UpdatePackages.test.js

```diff
@@ -0,0 +1,10 @@
+import { testComponentSnapshotsWithFixtures } from '../../../../../common/testHelpers';
+
+import UpdatePackages from '../../components/fields/UpdatePackages';
+
+import { updatePackagesProps } from '../fixtures'
+
+describe('RegistrationCommandsPage fields - UpdatePackages', () => {
+  testComponentSnapshotsWithFixtures(UpdatePackages, { 'renders': updatePackagesProps });
```

Comment:
not sure that snapshot testing is the right choice here, onlything I can imagine worth testing here is that the onChange method gets called.

The snapshot just tests that we render `CheckBox` component within it :)
I believe there is more value in not having the test. Can't imagine this snapshot to discover any true positive failure :)